extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -119,6 +126,21 @@ public WindowOperatorQueryFrameProcessor(
     for (int i = 0; i < frameReader.signature().size(); i++) {
       typeStrategies[i] = frameReader.signature().getColumnType(i).get().getNullableStrategy();
     }
+
+    // Get virtual columns to be added to the frame writer.
+    this.partitionBoostVirtualColumn = new SettableLongVirtualColumn(QueryKitUtils.PARTITION_BOOST_COLUMN);
+    final List<VirtualColumn> frameWriterVirtualColumns = new ArrayList<>();
+    if (addVirtualColumns) {
+      frameWriterVirtualColumns.add(partitionBoostVirtualColumn);
```

Comment:
Boosting isn't done since the query kit isn't setting the boost column. The boost column should be explicitly added in the window query kit for this line to take effect. We can take it up in a separate PR.

Reply:
Have removed the relevant lines.